deploy.sh

```diff
@@ -4,7 +4,7 @@
 mongod &
```

Comment:
May be worth moving all these .sh files into a `scripts/` folder. To declutter the root folder